### ISSUE TITLE
Closing the lid doesn't suspend the laptop

### ISSUE THREAD

**Issue by MarkkuR** [linuxmint/cinnamon#11477]
```
 * Cinnamon version 5.6.7
 * Distribution - Mint 21.1
 * Intel HD 520, default Intel driver
 * 64 bit
 * Logs don't display anything particular
 ```

### Issue

Closing the lid on a Dell Latitude E7270 doesn't suspend the machine or do anything else either.

### Steps to reproduce

Close the lid, machine stays on as usual.

### Expected behaviour

Machine should suspend (as it did on Mint 21 and earlier).

### Other information

Switching "Perform lid-closed action even with external monitors attached" (in Power Management settings) ON resolves the issue. Even after switching it back off suspend works as expected. If I may guess, there is an issue with the default settings which gets fixed after flicking the switch.

### Configuration info, if applicable

Brand new install from the latest ISO.

**Comment by anaximeno** [comment]
> Switching "Perform lid-closed action even with external monitors attached" (in Power Management settings) ON resolves the issue.

The problem happens even when you don't have external monitors active?

**Comment by MarkkuR** [comment]
> 

Yes. It was a clean new install and suspend never worked with the lid closed. Didn't even try plugging in a monitor.

**Comment by omian** [comment]
I've also been experiencing issues with suspend when lid is closed in Linux Mint 21.2 Cinnamon, even with no monitors attached. For me the issue is unreliable behaviour. Sometimes the laptop will suspend when I close the lid and sometimes it won't. The same issue occurred when I installed Ubuntu Cinnamon 22.04.2. However it didn't occur when I installed Ubuntu 23.04.3 LTS, Zorin OS 16.3, or Linux Mint 21.2 MATE. So it seems to me its related to something in Cinnamon. 

As per the original port, turning on "Perform lid-closed action even with external monitors attached" seems to resolve the issue (behaviour becomes reliable), at least it does in Linux Mint 21.2 Cinnamon. I didn't test this fix in Ubuntu Cinnamon 22.04.2.

The laptop I'm using is a MacBook Pro 2009, 13", Core 2 Duo 2.26 GHz, NVIDIA GeForce 9400M graphics.
